openedx/features/caliper_tracking/tests/current/edx.ui.lms.link_clicked.json

```diff
@@ -3,16 +3,16 @@
    "event_source":"browser",
    "name":"edx.ui.lms.link_clicked",
    "accept_language":"en-US,en;q=0.9,ur-PK;q=0.8,ur;q=0.7",
-   "time":"2018-10-16T14:23:24.785371+00:00",
+   "time":"2018-11-15T10:15:00.000Z",
    "agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
    "page":"http://localhost:18000/courses/course-v1:edx+cs-101+2018/course/",
```

Comment:
The value of `page` has already been used in the `object` value. so there is not need to put this in `extra_fields`